steps/genymotion-cloud-saas-start/0.0.1/step.yml

```diff
@@ -0,0 +1,92 @@
+title: Genymotion Cloud Start Android Devices
+summary: |
+  Start Genymotion Cloud SaaS Android Devices
+description: |-
+  This step launches Android devices on Genymotion Cloud SaaS.
+  Appium, Espresso, and others test automation frameworks can be run on those devices.
+  Setup instructions:
+  1. Create a Genymotion Cloud SaaS account: [https://cloud.geny.io](https://cloud.geny.io/?&utm_source=web-referral&utm_medium=docs&utm_campaign=bitrise&utm_content=signup)
+  1. Navigate to Default templates or templates:
+    - Select a device.
+    - Click on dot menu and select details.
+    - On the description field, get the recipe UUID.
+    - Copy paste it to use it in this step.
+website: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-start
+source_code_url: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-start
+support_url: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-start/issues
+published_at: 2020-01-06T17:33:28.928568045+01:00
+source:
+  git: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-start.git
+  commit: 577a9454bd58e9c7c5512e2d077b8eb3163a0073
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- utility
```

Comment:
It should be rather `test`

Reply:
@godrei said me to put utility as it does not perform any test by itself

Reply:
@koral-- I still keep it, it does not perform/execute any tests.

Reply:
There was a similar case and discussion with @godrei in 2017 and the conclusion was opposite.
https://github.com/bitrise-io/bitrise-steplib/pull/1101#issuecomment-314151310

Reply:
I see, at that time i stated the same as this case and i still keep it.
The source of this issue is that we did not have a proper guide which describes the project type tags.
As i mentioned previously the new Step Development Guideline is in progress, the Tech Writer team is working on the final touches.

Reply:
OK, thx.
If it is ready don't forget to check if existing steps are following new guidelines ;)